src/hooks/useAllServices.ts

```diff
@@ -24,7 +29,7 @@ const handleBundleResponse = (bundle: { data: Omit<BundleNavigation, 'id' | 'tit
             ...rest,
             navItems,
           },
-        }),
+        }).links,
```

Comment:
Can links be `undefined`?

Reply:
I don't think so, I can make it a safe accessor though.